playground/TestShop/TestShop.AppHost/Program.cs

```diff
@@ -6,11 +6,14 @@
                        .AddDatabase("catalogdb");
 
 var basketCache = builder.AddRedis("basketcache")
-                         .WithDataVolume()
-                         .WithRedisCommander(c =>
-                         {
-                             c.WithHostPort(33801);
-                         });
+                         .WithDataVolume();
+
+#if !SKIP_DASHBOARD_REFERENCE
+basketCache.WithRedisCommander(c =>
```

Comment:
indent

Reply:
The file has top-level statements, so zero indent. Or did you mean for the `{..}` block? Could you share the suggested indent in that case, and I can commit that?

Reply:
I can change in a follow up.